go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/gofrs/uuid v3.2.0+incompatible
 	github.com/google/go-cmp v0.5.9
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
+	github.com/hashicorp/golang-lru/v2 v2.0.5
```

Comment:
it looks like [v2.0.6](https://github.com/hashicorp/golang-lru/releases/tag/v2.0.6) has a bug-fix